app/posts/manage-your-appointments/2024/10/2024-10-10-designing-availability-view.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Designing the availability and bookings view"
+date: 2024-10-10
+---
+
+As part of managing availability and appointments, our users need to be able to view availability they’ve created, check how many appointments they have booked, and view a list of daily bookings for their site. 
+
+
+## What we did 
+
+From our discovery research, we know that users need to: 
+
+- See how many booked and unbooked appointments they had for each day, to help them understand demand for vaccinations. 
+- See what sessions and services they are providing for each day 
```

Comment:
I think these can both be lower-case with no full stop, as there’s a lead-in line?

```suggestion
- see how many booked and unbooked appointments they had for each day, to help them understand demand for vaccinations
- see what sessions and services they are providing for each day
```